### PR TITLE
STYLE: fix violation of the DRY principle

### PR DESCRIPTION
**Pull request by drveles**
## What was changed
The code has been updated in accordance with the [DRY principle](https://ru.wikipedia.org/wiki/Don’t_repeat_yourself).

## Why?
It is much smaller, more concise and faster to read.

1. Closes 

2. How was this tested:
Standard tests must pass.

3. Any docs updates needed?
No. The logic has not been changed.

### CODE REVIEW

**Review by cretz — APPROVED** [review]
Sometimes in samples we intentionally copy code as a demonstration of what a user might do if they had different code for each conditional. For example, if we really wanted to properly reuse code, we wouldn't have 4 separately named activities, we'd have one `order_fruit` activity that accepted a fruit type.

But I am ok with this. @dandavison - opinion?

**Comment by dandavison** [comment]
Nice cleanup @drveles. I agree -- while the activity calls are all the same it's appropriate for them to be written without repetition.

File: hello/hello_activity_choice.py [hunk 1]
```diff
@@ -59,39 +59,22 @@ async def run(self, list: ShoppingList) -> str:
         ordered: List[str] = []
         for item in list.items:
```
Comment (dandavison) [inline hello/hello_activity_choice.py:60]: A tiny thing, not related to this PR, but we should avoid using `list` as a variable name in Python.

**Comment by drveles** [comment]
Thank you. 
I will probably correct the code according to the remark about the `list'.

**Pull request merged**